core/src/main/java/com/google/zxing/oned/Code128Reader.java

```diff
@@ -482,9 +482,6 @@ public Result decodeRow(int rowNumber, BitArray row, Map<DecodeHintType,?> hints
                   }
                 }
                 break;
-              case CODE_FNC_2:
```

Comment:
I believe you, but why is this definitely unused?

Reply:
This `switch(code)` is in the `else` block from line 467, and corresponding `if` is at line 458.

The line 458 is:
```
if (code < 100) {
```

Therefore, the `code` in this `switch` expression is greater than or equal to 100.
And `CODE_FNC_2` is 97 (defined at line 160), so there is no chance to match to this `case`.

Reply:
OK you're right. The static analyzer agrees. @dehnhard you added this in #1372 - do you object?

Reply:
No objection from me. Looks like CODE_C is numeric only and can't contain FNC2 (but a leading FNC1). So, no harm done, it is removable dead code.
( and good to see improvements coming in on the symbology identifier code )